types/bitcoin-computer/index.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for @bitcoin-computer/lib 0.9
```

Comment:
should your types be under: `bitcoin-computer__lib` as per DT convention? (and when using generator):
https://github.com/DefinitelyTyped/DefinitelyTyped#what-about-scoped-packages

Reply:
Yes that would be better.